tests/test_batcheffect.yml

```diff
@@ -4,15 +4,15 @@
     - test_batcheffect
   files:
     - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_batch_b2_vs_b1.tsv
-      md5sum: 4c8bcede761f82803c262f28a13eb5ce
+    #      md5sum: 4c8bcede761f82803c262f28a13eb5ce
```

Comment:
Why were they uncommented? Are the files suddenly not producing the same md5sums anymore?

Reply:
Yep...they changed from time to time, I am not sure when exactly this happens, I believe in different PRs the 5sums change, almost as if the github server runner changes the files...

Reply:
How do the md5sums behave when you test locally? Are they stable over multiple runs?

Reply:
Testing this right now

Reply:
I cannot make head nor tail of that...On another branch, the 5sums are still enabled and it works. Locally, I got different sums when running the same profile on cfc and thanos. I'll reenable the 5sums here. If they again don't work, I'll probably have to delete them...